src/components/pagination.rs

```diff
@@ -31,6 +31,9 @@ pub struct PaginationProperties {
     /// additional styles
     #[prop_or_default]
     pub style: AttrValue,
+
+    #[prop_or_default]
+    pub is_bottom: bool,
```

Comment:
Thanks, I've replaced it by `pub enum PaginationPosition`. I didn't use `pub enum Position` because that enum has been defined already in other file and it caused conflicts. In any case, having the name of the component in the Enum name helps to understand the context of it.